src/Main.java

```diff
@@ -2,5 +2,103 @@ public class Main {
     public static void main(String[] args) {
         System.out.println("Hello, World!");
 
+        System.out.println("Задача№1");
+        int a = 4000;
+        byte b = 22;
+        float c = 3.75f;
+        short d = 25645;
+        long i = 9233567899L;
+        double f = -1.7E+308;
+        System.out.println("Значение переменной a с типом Целочисленныемравно " + a);
+        System.out.println("Значение переменной b с типом Целочисленныемравно " + b);
+        System.out.println("Значение переменной c с типом Целочисленныемравно " + c);
+        System.out.println("Значение переменной d с типом Целочисленныемравно " + d);
+        System.out.println("Значение переменной i с типом Целочисленныемравно " + i);
+        System.out.println("Значение переменной f с типом Целочисленныемравно " + f);
+        System.out.println("Задача№2");
+        float m = 27.12f;
+        long v = 987678965549L;
+        double x = 2.786;
+        short k = 569;
+        int t = - 259;
+        int p = 27897;
+        byte u = 67;
+        System.out.println(m);
+        System.out.println(v);
+        System.out.println(x);
+        System.out.println(k);
+        System.out.println(t);
+        System.out.println(p);
+        System.out.println(u);
+        System.out.println("Задача№3");
+        int classLP = 23;
+        int classAS = 27;
+        int classEA = 30;
+        int numberStudents = (int) (classLP + classAS + classEA);
+        int totalPaper = 480;
+        int paperStudents = totalPaper / numberStudents;
+        System.out.println("На каждого ученика расчитано " + paperStudents + " листов бумаги.");
+        System.out.println("Задача#4");
+        int time = 2;
+        int bottle = 16;
+        int bottleInMinute = (short) (bottle / time);
+        int day = (24 * 60);
+        int dayWeight =  (bottleInMinute * day);
+        int bottlesPer3Days = (1440 * 3);
+        int day2Weight =  (bottleInMinute * bottlesPer3Days);
+        int month =  (1440 * 30);
```

Comment:
скобки не нужны + надо использовать переменные
```suggestion
        int month =  day * 30;
```